agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/AlleleRepository.java

```diff
@@ -19,7 +19,7 @@ public Allele getAllele(String primaryKey) {
 
         map.put("primaryKey", primaryKey);
         String query = "";
-        query += " MATCH p1=(:Species)-[:FROM_SPECIES]-(a:Allele) WHERE a.primaryKey = {primaryKey}";
+        query += " MATCH p1=(:Species)-[:FROM_SPECIES]-(a:Allele)--(:CrossReference) WHERE a.primaryKey = {primaryKey}";
         query += " OPTIONAL MATCH p3=(a:Allele)-[:IS_ALLELE_OF]-(gene:Gene)-[:FROM_SPECIES]-(:Species)";
         query += " OPTIONAL MATCH p4=(a:Allele)-[:ALSO_KNOWN_AS]-(:Synonym)";
         query += " OPTIONAL MATCH crossRef=(a:Allele)-[:CROSS_REFERENCE]-(:CrossReference)";
```

Comment:
Won't need the optional match plus you might want to add [:CROSS_REFERENCE] between Allele and CrossRef